### ISSUE TITLE
Missing location in response header

### ISSUE THREAD

**Issue by orhantoy** [opbeat/opbeat_ruby#23]
It seems as this line https://github.com/opbeat/opbeat_ruby/blob/master/lib/opbeat/client.rb#L90 fails because `location` is missing from the response header. An example of the returned response header verifies this:

```json
{
    "connection": "Close",
    "content-length": "57",
    "content-type": "application/json; charset=UTF-8",
    "date": "Sun, 01 Feb 2015 12:47:34 GMT",
    "server": "nginx",
    "x-ratelimit-limit": "250",
    "x-ratelimit-remaining": "249",
    "x-ratelimit-reset": "2015-02-01T12:48:34.112711Z"
}
```

**Comment by orhantoy** [comment]
I forgot to mention that the above response is from a new **release**. A new **event** returns the following response headers, which correctly includes `location`:

```json
{
    "connection": "Close",
    "content-length": "55",
    "content-type": "application/json; charset=UTF-8",
    "date": "Sun, 01 Feb 2015 13:00:56 GMT",
    "location": "https://opbeat.com/xxxxxxxxxxx",
    "server": "nginx",
    "x-ratelimit-limit": "250",
    "x-ratelimit-remaining": "249",
    "x-ratelimit-reset": "2015-02-01T13:01:56.043741Z"
}
```

**Comment by watson** [comment]
@orhantoy Thanks for debugging and reporting this bug :smiley: It's fixed and released in v1.1.1 :shipit:
